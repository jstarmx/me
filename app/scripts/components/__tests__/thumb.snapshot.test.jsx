import React from 'react';
import renderer from 'react-test-renderer';

import Thumb from '../thumb';

const props = { path: 'path', thumb: 'thumb', title: 'title' };

it('renders a Thumb component', () => {
  const component = renderer.create(<Thumb { ...props } />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
