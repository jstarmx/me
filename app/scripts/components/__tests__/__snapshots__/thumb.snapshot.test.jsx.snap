// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a Thumb component 1`] = `
<li
  className="thumb"
  style={
    Object {
      "backgroundImage": "url(\\"thumb\\")",
    }
  }
>
  <a
    className="thumb__link"
    href="path"
    onClick={[Function]}
    rel="noopener noreferrer"
    target="_blank"
  >
    <span
      className="thumb__title"
    >
      title
    </span>
  </a>
</li>
`;
